testsuite/features/build_validation/retail/sle12sp5_buildhost_build_kiwi_image.feature

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2021-2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@sle12sp5_buildhost
+Feature: Prepare buildhost and build OS image for SLES 12 SP5
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Create the bootstrap repository for a SLES 12 SP5 build host
+     When I create the bootstrap repository for "sle12sp5_buildhost" on the server
+
+  Scenario: Clean up sumaform leftovers on a SLES 12 SP5 build host
+    When I perform a full salt minion cleanup on "sle12sp5_buildhost"
+
+  Scenario: Bootstrap the SLES 12 SP5 build host
+    When I bootstrap minion client "sle12sp5_buildhost" using bootstrap script with activation key "1-sle12sp5_buildhost_key" from the proxy
+    And I wait at most 10 seconds until Salt master sees "sle12sp5_buildhost" as "unaccepted"
+    And I accept "sle12sp5_buildhost" key in the Salt master
+    And I wait until onboarding is completed for "sle12sp5_buildhost"
+
+  Scenario: Turn the SLES 12 SP5 build host into a OS image build host
+    Given I am on the Systems overview page of this "sle12sp5_buildhost"
+    When I follow "Details" in the content area
+    And I follow "Properties" in the content area
+    And I check "osimage_build_host"
+    And I click on "Update Properties"
+    Then I should see a "OS Image Build Host type has been applied." text
+
+  Scenario: Apply the highstate to the SLES 12 SP5 build host
+    Given I am on the Systems overview page of this "sles12sp5_buildhost"
+    When I wait until no Salt job is running on "sles12sp5_buildhost"
+    And I apply highstate on "sles12sp5_buildhost"
+    And I wait until file "/var/lib/Kiwi/repo/rhn-org-trusted-ssl-cert-osimage-1.0-1.noarch.rpm" exists on "sles12sp5_buildhost"
+
+  Scenario: Create an OS image profile for SLES 12 SP5 with activation key
+    When I follow the left menu "Images > Profiles"
+    And I follow "Create"
+    And I enter "suse_os_image_12" as "label"
+    And I select "Kiwi" from "imageType"
+    And I select "1-sle12sp5_minion_key" from "activationKey"
+    And I enter the image filename for "sle12sp5_terminal" relative to profiles as "path"
+    And I click on "create-btn"
+
+  # WORKAROUND
+  # Remove as soon as the issue is fixed
+  Scenario: Work around issue https://github.com/SUSE/spacewalk/issues/10360
+    When I let Kiwi build from external repositories
+
+  Scenario: Build Kiwi image for SLES 12 SP5
+    When I follow the left menu "Images > Build"
+    And I select "suse_os_image_12" from "profileId"
+    And I select the hostname of "sle12sp5_buildhost" from "buildHostId"
+    And I click on "submit-btn"
+
+  Scenario: Check the OS image built
```

Comment:
```suggestion
  Scenario: Check the OS image build
```
or
```suggestion
  Scenario: Check that the OS image built
```

Reply:
Nope, it checks the image, and the image has been built.

If you really don't like it, I can use `Check the OS image just built" or `Check the image that has been built`.